Consumer/src/ApiClient.cs

```diff
@@ -2,47 +2,42 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 
-namespace Consumer
+namespace Consumer;
+
+public class ApiClient
 {
-    public class ApiClient
+    private readonly Uri _baseUri;
+
+    public ApiClient(Uri baseUri)
     {
-        private readonly Uri BaseUri;
+        _baseUri = baseUri;
```

Comment:
the change `_baseUri = baseUri` is not reflected in the docs which still use `private readonly Uri BaseUri;`

updating the code snippet in the readme is an easy copy pasta